src/models/user.js

```diff
@@ -0,0 +1,34 @@
+import { Model } from 'sequelize';
+
+export default (sequelize, DataTypes) => {
+  class User extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    // We might or might not need this association depending on the design.
+    // Commenting out right now to satisfy the linter.
+    // static associate(models) {
+    //   // define association here
+    // }
+  }
+  User.init({
+    id: {
+      type: DataTypes.BIGINT,
```

Comment:
Does it matter that this is a `BIGINT` here and an `INTEGER` in the migration?

Reply:
Will adjust from `BIGINT` to `INTEGER`. Given the number of users it doesn't look like we will ever need `BIGINT ` here.

Reply:
I'd vote for making them both `BIGINT`. Storage is cheap and it's one less thing to worry about a couple years into the future. Unless there's a compelling reason for `INTEGER`?

Reply:
If we use INTEGER, that would give us 2,147,483,647 ids. I think that's enough.